### DISCUSSION TITLE
How to send continuous Slack alerts?

### DISCUSSION THREAD

**Discussion by mani9223-oss** [TwiN/gatus#296] [Q&A] [answered]
@TwiN Is it possible to send continuous Slack alerts for failures? Right now it is sending an alert one time and sends another one only after it's resolved. It would be good to continuously send alerts for failures until its resolved. 

Your thoughts, please...

**Comment by TwiN** [comment, marked as answer]
That's not currently possible, and it's by design: https://github.com/TwiN/gatus/blob/829a9c26799a946b8788bf76497cf26c6b1dc2a8/watchdog/alerting.go#L32-L37

The reason behind this choice is that spam tends to lead to people ignoring alerts, and I built Gatus in a way that minimizes the number of alert to keep them meaningful.

That being said, if that's a feature you're interested in, I wouldn't be against adding something like `alert.trigger-repeatedly`.
